From a81b4f95d30b29ea8ab0fcdbb0032fc34c3e9fc2 Mon Sep 17 00:00:00 2001
From: Sergio Costas <raster@rastersoft.com>
Date: Sat, 15 Dec 2018 23:35:50 +0100
Subject: [PATCH 54/55] desktopGrid.js: Use Unicode apostrophe

Two log strings used an escaped ASCII apostrophe, but it still is
incorrectly recognized by xgettext, which shows a warning. This
patch changes it with the Unicode one, fixing this problem.
---
 fileItem.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fileItem.js b/fileItem.js
index 9fd6fc8..05db8fa 100644
--- a/fileItem.js
+++ b/fileItem.js
@@ -238,7 +238,7 @@ var FileItem = class {
         if (this._isDesktopFile) {
             this._desktopFile = Gio.DesktopAppInfo.new_from_filename(this._file.get_path());
             if (!this._desktopFile) {
-                log(`Couldn\'t parse ${this._displayName} as a desktop file, will treat it as a regular file.`);
+                log(`Couldn’t parse ${this._displayName} as a desktop file, will treat it as a regular file.`);
                 this._isDesktopFile = false;
             }
         }
@@ -417,7 +417,7 @@ var FileItem = class {
 
     doOpen() {
         if (this._isBrokenSymlink) {
-            log(`Error: Can\'t open ${this.file.get_uri()} because it is a broken symlink.`);
+            log(`Error: Can’t open ${this.file.get_uri()} because it is a broken symlink.`);
             return;
         }
 
-- 
2.20.0

